Ryujinx.Debugger/Ryujinx.Debugger.csproj

```
@@ -0,0 +1,42 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <Configurations>Debug;Release;Profile Release;Profile Debug</Configurations>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Profile Debug|AnyCPU'">
+    <DefineConstants>TRACE;USE_DEBUGGING</DefineConstants>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Profile Release|AnyCPU'">
+    <DefineConstants>TRACE;USE_DEBUGGING</DefineConstants>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Remove="UI\DebbugerWidget.glade" />
```

```suggestion
    <None Remove="UI\DebuggerWidget.glade" />
```
And rename the file too.